components/sandbox/variables.tf

```diff
@@ -3,3 +3,8 @@ variable "resource_group_name" {
   description = "The name of the resource group that contains the DNS zone where the records will be added."
   default     = "reformmgmtrg"
 }
+
+variable "env" {
```

Comment:
why do you have this? doesn't seem to be used?

Reply:
removed it